authz/rbac_translator.go

```diff
@@ -302,14 +302,10 @@ func (options *auditLoggingOptions) toProtos() (allow *v3rbacpb.RBAC_AuditLoggin
 		deny.AuditCondition = toDenyCondition(v3rbacpb.RBAC_AuditLoggingOptions_AuditCondition(rbacCondition))
 	}
 
-	for i := range options.AuditLoggers {
-		config := &options.AuditLoggers[i]
-		if config.Config == nil {
-			return nil, nil, fmt.Errorf("AuditLogger Config field cannot be nil")
-		}
+	for _, config := range options.AuditLoggers {
 		typedStruct := &v1xdsudpatypepb.TypedStruct{
-			TypeUrl: typedURLPrefix + config.Name,
-			Value:   config.Config,
+			TypeUrl: typeURLPrefix + config.Name,
```

Comment:
Did we use to check the non-emptiness of `config.Name`? We seem to be relying on the `TypeUrl` causing the logger builder not found here. Would it make more sense to reject empty or missing `name` field directly? For reference: https://github.com/grpc/grpc/blob/1624542ea428688efcd31b0edc0b52fee948ae68/src/core/lib/security/authorization/rbac_translator.cc#L391

Reply:
Addressed, PTAL